### ISSUE TITLE
Make it easier to find Configuration documentation

### ISSUE THREAD

**Issue by DanKaplanSES** [jasmine/jasmine.github.io#161]
In my humble opinion it's challenging to find where the configuration file is documented from the documentation page (note the red):

![image](https://github.com/jasmine/jasmine.github.io/assets/373675/6a64a4d6-36df-445a-bbd7-6231ecd09084)

I've used Jasmine for years, so the "Getting Started" section is not the place I expect to find the reference documentation for configuration files. That said, I understand the value in putting it there because configuring Jasmine is an early step in using it.

I propose prefacing these two links with the word, "Configuring".

**Comment by sgravrock** [comment]
I agree that configuration docs are hard to find, but I don't think changing the text of those links is the right solution. The biggest problem is that those pages aren't just about configuration. Changing the titles to e.g. "Configuring Node.js" might very well steer people who are looking for a general getting-started guide away. The other problem is that those pages don't comprehensively document configuration. For instance, the Node getting started page only mentions a handful of the most common core config properties, and the browser getting started page doesn't discuss core configuration at all.

I think what we really need is an additional configuration guide page that goes into more depth about configuration and provides links to the relevant API reference docs for jasmine-core, jasmine-browser-runner, and jasmine-npm configuration. That's on my to-do list. But it's not at the top of my to-do list yet, so a PR would certainly be welcome if you or anyone else feels like taking that on.

**Comment by DanKaplanSES** [comment]
> The biggest problem is that those pages aren't just about configuration. Changing the titles to e.g. "Configuring Node.js" might very well steer people who are looking for a general getting-started guide away.

Makes sense. What about this as a change:

![image](https://github.com/jasmine/jasmine.github.io/assets/373675/a62e6aa4-5975-426b-91dd-24030418ab23)

> a PR would certainly be welcome if you or anyone else feels like taking that on

Unfortunately, I don't think I know enough about those topics to write the guide myself.

**Comment by DanKaplanSES** [comment]
@sgravrock Sorry for the bump, but do you have any thoughts on my previous comment? 

That image is meant as an increment to improve the current state, not an _alternative_ to your idea. I think your idea is great, I just don't have the knowledge implement it; whereas, if you see this image as an improvement on the current state, I think I can create a PR for it without requiring a brain dump from somebody as a prerequisite.

**Comment by sgravrock** [comment]
I'm not sure I fully understand. Are you proposing to break up the existing Getting Started pages into per-section pages and have each of those links go to one of the per-section pages? Or keep the pages as-is and have each of those links go to anchors within the existing pages? Either of those approaches seems reasonable to me.
